astroquery/gaia/core.py

```diff
@@ -1168,23 +1168,19 @@ def get_status_messages(self):
         """Retrieve the messages to inform users about
         the status of Gaia TAP
         """
-        try:
-            sub_context = self.GAIA_MESSAGES
-            conn_handler = self._TapPlus__getconnhandler()
-            response = conn_handler.execute_tapget(sub_context, verbose=False)
-            if response.status == 200:
-                if isinstance(response, Iterable):
-                    for line in response:
-
-                        try:
-                            print(line.decode("utf-8").split('=', 1)[1])
-                        except ValueError as e:
-                            print(e)
-                        except IndexError:
-                            print("Archive down for maintenance")
-
-        except OSError:
```

Comment:
This exception catching is a particular problem: it hid from the tests that there is a remote query being performed, since pytest-remotedata raises an OSError